src/pages/pool/_id.vue

```diff
@@ -3,116 +3,20 @@
-            <APRTooltip :pool="pool" class="-ml-1 mt-1" />
-            <BalLink
-              :href="explorer.addressLink(pool?.address || '')"
-              external
-              noStyle
-              class="flex items-center"
-            >
-              <BalIcon
-                name="arrow-up-right"
-                size="sm"
-                class="ml-2 mt-2 text-gray-500 hover:text-blue-500 transition-colors"
-              />
-            </BalLink>
-          </div>
-          <div class="flex items-center mt-2">
-            <div v-html="poolFeeLabel" class="text-sm text-secondary mr-1" />
-            <BalTooltip>
-              <template v-slot:activator>
-                <BalLink
-                  v-if="feesManagedByGauntlet"
-                  :href="EXTERNAL_LINKS.Gauntlet.Home"
-                  external
-                >
-                  <GauntletIcon />
-                </BalLink>
-                <BalIcon
-                  v-else
-                  name="info"
-                  size="xs"
-                  class="text-gray-400 dark:text-gray-500"
-                />
-              </template>
-              <span>
-                {{ swapFeeToolTip }}
-              </span>
-            </BalTooltip>
-          </div>
-        </div>
-
-        <BalAlert
-          v-if="!appLoading && !loadingPool && missingPrices"
-          type="warning"
-          :title="$t('noPriceInfo')"
-          class="mt-2"
-          block
-        />
-        <BalAlert
-          v-if="!appLoading && !loadingPool && hasCustomToken"
-          type="error"
-          :title="$t('highRiskPool')"
-          class="mt-2"
-          block
-        />
-        <template v-if="!appLoading && !loadingPool && isAffected">
-          <BalAlert
-            v-for="(warning, i) in warnings"
-            :key="`warning-${i}`"
-            type="error"
-            class="mt-2"
-            block
-          >
-            <template #title>
-              <div v-html="warning.title" />
-            </template>
-            <template #description>
-              <div v-html="warning.description" />
-            </template>
-          </BalAlert>
-        </template>
-        <BalAlert
-          v-if="!appLoading && noInitLiquidity"
-          type="warning"
-          :title="$t('noInitLiquidity')"
-          :description="$t('noInitLiquidityDetail')"
-          class="mt-2"
-          block
-        />
-      </div>
-
+      <PoolPageHeader
+        :loadingPool="loadingPool"
+        :poolTypeLabel="poolTypeLabel"
+        :loadingApr="loadingApr"
+        :pool="pool"
+        :poolApr="poolApr"
+        :isStableLikePool="isStableLikePool"
+        :noInitLiquidity="noInitLiquidity"
+        :titleTokens="titleTokens"
+        :missingPrices="missingPrices"
+        :isLiquidityBootstrappingPool="isLiquidityBootstrappingPool"
+        :isStablePhantomPool="isLiquidityBootstrappingPool"
```

Comment:
Can a bunch of these props not be removed by moving the computed properties into the PoolPageHeader component?

Reply:
moved one to `PoolPageHeader`, others are used in `_id.vue` file